packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -332,7 +333,9 @@ async function fetchLatestSnapshotCore(
     });
 }
 
-function evalBlobsAndTrees(snapshot: IOdspSnapshot) {
+function validateAndEvalBlobsAndTrees(snapshot: IOdspSnapshot) {
+    assert(Array.isArray(snapshot.trees), "Returned odsp snapshot is malformed. No trees!");
+    assert(Array.isArray(snapshot.blobs), "Returned odsp snapshot is malformed. No blobs!");
```

Comment:
I think same as trees we would have at least 1 blob. So asserting for blob also.